backup/queries_statistics.go

```diff
@@ -119,14 +119,14 @@ func GetAttributeStatistics(connectionPool *dbconn.DBConn, tables []Table) map[u
 		inheritClause, statSlotClause, statCollationClause,
 		SchemaFilterClause("n"), utils.SliceToQuotedString(tablenames))
 
-	results := make([]AttributeStatistic, 0)
-	err := connectionPool.Select(&results, query)
-	gplog.FatalOnError(err)
-	stats := make(map[uint32][]AttributeStatistic)
-	for _, stat := range results {
-		stats[stat.Oid] = append(stats[stat.Oid], stat)
+	rows, err := connectionPool.Query(query)
```

Comment:
Here `err` value is actually not used, as it is overwritten in the loop before first usage.

Reply:
fixed